wagtailio/static/sass/components/_cta.scss

```diff
@@ -42,13 +46,16 @@
         > svg {
             fill: $color--white;
             transform: translateX(10px) scale(1.1);
+
```

Comment:
I think repeating this rule on hover is unnecessary though

Reply:
Because the arrow icon is white when hovered, it practically disappears for contrast themes with light backgrounds. That's why I chose to repeat the rule here.

Reply:
good point!